benchmark/src/main.rs

```diff
@@ -0,0 +1,20 @@
+use anyhow::Result;
+use benchmark::{Benchmark, CommandRunner};
+use clap::Parser;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    env_logger::builder()
+        .filter_module("benchmark", log::LevelFilter::Info)
+        .init();
+
+    let args: Benchmark = Benchmark::parse();
+
+    let mut runner = CommandRunner::new(args);
+    let stats = runner.run().await?;
+
+    println!("{}", stats.summary());
```

Comment:
Please use log as you've init it at the beginning of the function.

Reply:
We need to output these information to stdout, log is only used to observe the running status of the program